terraso_backend/apps/auth/services.py

```diff
@@ -48,6 +54,14 @@ def _persist_user(self, email, first_name="", last_name=""):
         if update_name:
             user.save()
 
+        user_id = str(user.id)
+        try:
+            has_picture = profile_image_service.exists(user_id)
+            if not has_picture and profile_image_url:
+                profile_image_service.upload_url(user_id, profile_image_url)
+        except Exception:
+            logger.exception("Failed to upload profile image")
```

Comment:
We don't log exceptions anywhere else. If we are going to log things, it would be good to do so (a) with rollbar and (b) more broadly. In this specific case, we should include the user ID as part of the message.

For now, have this return `None`. I opened a new ticket to handle error logging.